src/helpers/button/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import "./Button.css"
+
+function Button(props) {
```

Comment:
Je zou de props kunnen spreaden, zodat je niet overal `props.` vóór hoeft te zetten
```suggestion
function Button({ classname, type, id, name, onClick, text }) {
```